assets/js/components/VehicleStatus.vue

```diff
@@ -422,7 +422,7 @@ export default {
 			});
 		},
 		smartCostVisible() {
-			return !!this.smartCostLimit;
```

Comment:
Lässt sich in JS tatsächlich nicht auf Identität mit null vergleichen?

Reply:
Doch, aber dann müsste ich `null` und `undefined` prüfen. Wäre nicht wirklich kürzer. Daher hab ich hier die "Prüfung auf Nummer" gemacht.

Reply:
Die wunderbare JS Welt. Wieder was gelernt- Danke 🙏